### PR TITLE
Hub admin policies

### PR DESCRIPTION
**Pull request by ateucher**
Restructure organization of policies/workshop setup, and add policies from https://github.com/NASA-Openscapes/2i2cAccessPolicies

@jules32 I this is ready for a review. I created a new section "Policies and Administration", and put inside it:

- index.qmd containing the policies from https://github.com/NASA-Openscapes/2i2cAccessPolicies, as well as the previous material about cost explorer and budget setup etc.
- add-folks-to-2i2c-github-teams.qmd - The newly revamped instructions from #326 
- data-policies.qmd - the data policies and HOME dir usage guidance from https://github.com/NASA-Openscapes/2i2cAccessPolicies
- leading-workshops.qmd (largely unchanged)

To do:
- Create policies and write them up for the "just-in-time" single password workshop hub and revamp leading-workshops.qmd
- remove policies from README at https://github.com/NASA-Openscapes/2i2cAccessPolicies and link to here
- Flesh out the cost explorer/budget stuff, maybe put it in its own page, and link to cost report (when it's ready)

### CODE REVIEW

File: policies-admin/index.qmd [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+### Obtaining Access to the NASA Openscapes Hub
+
+Access is controlled by the NASA Openscapes Team, who oversee the management of the Hub and Cloud costs. The first step to gaining access to the NASA Openscapes 2i2c Hub is to request access via [this form](https://forms.gle/sLM9szAYN2mq6SbL9). 
+
+Our JupyterHub users are managed in three GitHub Teams: 
+
+* [Long-term access](https://github.com/orgs/nasa-openscapes-workshops/teams/longtermaccess-2i2c): This access is for NASA Openscapes mentors and team, DAAC staff and others who request a longer-term engagement  
+* NASA Openscapes Champions: This access is for teams that participate in the NASA Openscapes Champions Program. These teams have access for up to a year as they migrate their workflows to the Cloud. 
+* Workshops and Hack-a-thons: This provides short term access of up to 1-month to participants of NASA Earthdata workshops. Participants will be removed at any time and have no expectation of on-going storage in their home directories.
```
Comment (jules32) [inline policies-admin/index.qmd:29]:
```suggestion
* Workshops and Hackathons: This provides short term access of up to 1-month to participants of NASA Earthdata workshops. Participants will be removed at any time and have no expectation of on-going storage in their home directories.
```

File: policies-admin/index.qmd [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Instructions for admins on how to add people to the hub can be found [here](add-folks-to-2i2c-github-teams.qmd).
+
+### Allowable Uses of 2i2c Hub 
+
+Users who join these GitHub teams agree to use the NASA Openscapes Hub only for work on NASA EarthData related activities. Generally, recommended instance size is the smallest instance (1.9GB RAM and up to 3.75 CPUs). 
+
+Run large or parallel jobs over large geographic bounding boxes or over long temporal extents should be cleared with the NASA Openscapes Team by submitting an issue to this repo.
```
Comment (jules32) [inline policies-admin/index.qmd:37]:
```suggestion
Run large or parallel jobs over large geographic bounding boxes or over long temporal extents should be cleared with the NASA Openscapes Team by submitting an issue to the [2i2cAccessPolicies repository]( https://github.com/NASA-Openscapes/2i2cAccessPolicies).   
```

**Review by jules32 — APPROVED** [review]

**Comment by jules32** [comment]
Hi Andy I think this looks good and I like the next steps as well. You're welcome to merge this and then carry on with the other TODOs as well! Thank you!

**Comment by ateucher** [comment]
Thanks for the fast review!

**Pull request merged**